### PR TITLE
Nn 3359

### PR DESCRIPTION
**Pull request by JamieDraperUK**
- Apply new formatting to key worker lists in the rest of the pages
- Remove formatting duplication

### CODE REVIEW

File: backend/controllers/keyworkerShared.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { putLastNameFirst } = require('../utils')
+
+const sortAndFormatKeyworkerNameAndAllocationCount = (keyworkers) => {
+  return keyworkers
```
Comment (jsrobertson) [inline backend/controllers/keyworkerShared.js:4]: Don't need the `return`

**Review by jsrobertson — APPROVED** [review]

File: backend/tests/searchOffendersController.test.js [hunk 2]
```diff
@@ -181,11 +181,11 @@ describe('Search offenders controller', () => {
                 value: '1:G0276VC:true',
```
Comment (sp-watson) [inline backend/tests/searchOffendersController.test.js:181]: Really there ought to be a test to cover the ordering.

File: backend/tests/viewResidentialLocationController.test.js [hunk 3]
```diff
@@ -352,7 +352,7 @@ describe('View residential location', () => {
                 isHighComplexity: false,
```
Comment (sp-watson) [inline backend/tests/viewResidentialLocationController.test.js:352]: Really there ought to be a test to cover the ordering.

File: backend/controllers/allocateKeyWorker.js [hunk 4]
```diff
@@ -80,13 +74,10 @@ module.exports = ({ allocationService, keyworkerApi, oauthApi }) => {
             isManualAllocation &&
             `${offender.keyworkerDisplay} ${formatNumberAllocated(offender.numberAllocated)}`,
           keyworkerStaffId: staffId,
-          keyworkerList: sortedSelectableKeyworkers.map((keyworker) => {
```
Comment (sp-watson) [inline backend/controllers/allocateKeyWorker.js:83]: hmm - are there no tests for this page? Maybe would be good to add ones?

**Review by sp-watson — APPROVED** [review]

File: backend/controllers/allocateKeyWorker.js (hunk 4 above)
Comment (JamieDraperUK) [reply backend/controllers/allocateKeyWorker.js:83]: There are tests from the previous PR. This change is none functional.

File: backend/tests/searchOffendersController.test.js (hunk 2 above)
Comment (JamieDraperUK) [reply backend/tests/searchOffendersController.test.js:181]: Agreed. I've pulled those tests out into keyworkerShared.test.js

**Review by sp-watson — APPROVED** [review]

**Pull request merged**